src/main/java/com/wojciechbarwinski/demo/epic_board_games_shop/OrderCronJob.java

```diff
@@ -0,0 +1,51 @@
+package com.wojciechbarwinski.demo.epic_board_games_shop;
+
+import com.wojciechbarwinski.demo.epic_board_games_shop.entities.Order;
+import com.wojciechbarwinski.demo.epic_board_games_shop.entities.OrderStatus;
+import com.wojciechbarwinski.demo.epic_board_games_shop.repositories.OrderRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Component
+@RequiredArgsConstructor
+public class OrderCronJob {
+
+    private final OrderRepository orderRepository;
+
+    @Scheduled(cron = "0 */10 * * * * ") //every 10 minutes
```

Comment:
Cron schedule powinno być w propertasach (generalnie te dane, które sterują tym co ile coś się wykonuje - to może być cron expression, jak tutaj, ale to może być też np. int mówiący co ile ma się coś wykonać